ui/src/index.tsx

```diff
@@ -349,7 +352,9 @@ class Root extends PureComponent {
                             </Route>
                           </Route>
                           <Route path="settings">
-                            <IndexRoute component={MembersIndex} />
+                            <CloudOnly><IndexRoute component={VariablesIndex} /></CloudOnly>
+                            <CloudExclude><IndexRoute component={MembersIndex} /></CloudExclude>
+
                             <Route path="members" component={MembersIndex}>
```

Comment:
hrrm. I kinda think we should place all of the `/members` route behind `CloudExclude` so that users can't manually route to it either?